setup.py

```diff
@@ -5,21 +5,17 @@
 import versioneer
 
 
-here = os.path.abspath(os.path.dirname(__file__))
-packages = []
-for d, _, _ in os.walk(os.path.join(here, 'nc_time_axis')):
-    if os.path.exists(os.path.join(d, '__init__.py')):
-        packages.append(d[len(here)+1:].replace(os.path.sep, '.'))
-
 setup_args = dict(
     name='nc-time-axis',
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
     description='netcdftime support for matplotlib axis',
+    license='BSD3',
     author='Philip Elson',
-    author_email='pelson@users.noreply.github.com',
     url='https://github.com/scitools/nc-time-axis',
-    packages=packages,
+    package_dir = {'': 'lib'},
+    packages=['nc_time_axis'],
```

Comment:
@lbdreyer You need to be careful here ... If the package grows (maybe, maybe not) then subsequent package additions will not be included.